jenkins_pipelines/environments/common/pipeline-pull-request-light.groovy

```diff
@@ -0,0 +1,208 @@
+                            }
+                        }
+                    }
+                }
+            }
+            stage('Get test results') {
+                if(environment_workspace && common_params != ''){
+                    ws(environment_workspace){
+                        def error = 0
+                        if(must_test) {
+                            if (deployed) {
+                                try {
+                                    sh "./terracumber-cli ${common_params} --logfile ${resultdirbuild}/testsuite.log --runstep cucumber --cucumber-cmd 'cd /root/spacewalk/testsuite; rake cucumber:finishing_pr'"
+                                } catch(Exception ex) {
+                                    println("ERROR: rake cucumber:finishing_pr failed")
+                                    error = 1
+                                }
+                                try {
+                                    sh "./terracumber-cli ${common_params} --logfile ${resultdirbuild}/testsuite.log --runstep cucumber --cucumber-cmd 'cd /root/spacewalk/testsuite; rake utils:generate_test_report'"
+                                } catch(Exception ex) {
+                                    println("ERROR: rake utils:generate_test_repor failed")
+                                    error = 1
+                                }
+                                sh "./terracumber-cli ${common_params} --logfile ${resultdirbuild}/testsuite.log --runstep getresults"
+                                publishHTML( target: [
+                                        allowMissing: true,
+                                        alwaysLinkToLastBuild: false,
+                                        keepAll: true,
+                                        reportDir: "${resultdirbuild}/cucumber_report/",
+                                        reportFiles: 'cucumber_report.html',
+                                        reportName: "TestSuite Report for Pull Request ${builder_project}:${pull_request_number}"]
+                                )
+                                junit allowEmptyResults: true, testResults: "results/${BUILD_NUMBER}/results_junit/*.xml"
+                            }
+                            if (fileExists("results/${BUILD_NUMBER}")) {
+                                archiveArtifacts artifacts: "results/${BUILD_NUMBER}/**/*"
+                            }
+                            if (email_to != '') {
+                                sh " export TF_VAR_MAIL_TO=${email_to};export TF_VAR_URL_PREFIX=${url_prefix}; ./terracumber-cli ${common_params} --logfile ${resultdirbuild}/mail.log --runstep mail"
+                            }
+                            // Clean up old results
+                            sh "./clean-old-results -r ${resultdir} -s 10"
```

Comment:
I wonder if this is correct.
We have 10 environments, but we also specify that we want to keep for 15 days all the jobs executed. If we clean old results for a max of 10, then... we might end removing results to early.